Constants.php

```diff
@@ -193,6 +193,7 @@ function ChainLinksPieces()
 //5 - Hit on link
 //6 - Modified Base Attack Stats - e.g. Transmogrify
 //7 - Modal Play Ability - e.g. Enlightened Strike
+//8 - Defending Cards

```

Comment:
You shouldn't need to add defending cards to chainLinkSummary, there's already a $chainLinks that stores the cards still on the combat chain (see the above function)

Reply:
I never realized the frame for this kind of interactions was already implemented. I thought it needed to be started from the ground up, considering it was never implemented to work 100%. 

I've removed the info from chainLinkSummary and use ChainLinks properly